src/vmm/src/devices/virtio/vsock/mod.rs

```diff
@@ -138,6 +139,8 @@ pub enum VsockError {
     VirtioState(VirtioStateError),
     /// Vsock uds backend error: {0}
     VsockUdsBackend(VsockUnixBackendError),
+    /// Underlying IovDeque error: {0}
+    IovDeque(IovDequeError),
```

Comment:
if you do `IovDeque(#[from] IovDequeError)` you can avoid the `map_err` later on.

Reply:
I know, but other error variants do not use `#[from]` so I didn't use it. Also `#[from]` makes code less clear as you won't know what type of error is returned.